pdsview/labelError.py

```diff
@@ -0,0 +1,35 @@
+from ginga.qtw.QtHelp import QtGui, QtCore
+
+
+class LabelError(QtGui.QDialog):
```

Comment:
Where does this get used?  What role does it have, I don't see it being used anywhere or is it used implicitly somehow?